templates/pages/amp/category.html

```diff
@@ -28,13 +28,13 @@ <h1 class="categoryView-title">
             {{category.name}}
         </h1>
     </div>
-    <main class="page-content" id="product-listing-container">
+    <div class="page-content" id="product-listing-container">
```

Comment:
Could you please explain why do we need to remove main tag here?

Reply:
Could you please double check this page and point to the place with 2nd main tag (I probably skipped it)

Reply:
<img width="1386" alt="duplicated_tags" src="https://user-images.githubusercontent.com/67792608/114555845-c4629f80-9c70-11eb-9fb1-25e65d7c3945.png">

Reply:
screen is attached. It's easier to see through it

Reply:
See, thanks